ProcessLib/ProcessVariable.h

```diff
@@ -75,10 +75,9 @@ class ProcessVariable
 		}
 	}
 
-	template <typename OutputIterator, typename GlobalSetup, typename... Args>
+	template <typename GlobalSetup, typename OutputIterator, typename... Args>
```

Comment:
Not more and not less than before this PR. Within our current usage, we can easily provide GlobalSetup solely as global typedef. I think that would be a good thing, but it would would introduce changes in quite some places (I guess; easy changes, but at many places). So I'd like to leave that change for a future PR.